docs/SMCWG-charter.md

```diff
@@ -0,0 +1,328 @@
+cases are not in scope for the SMCWG, care will be exercised by the SMCWG to
+avoid unintended adverse effects to these uses. The security, stability and
+resiliency of the Internet shall be taken into consideration when the SMCWG
+forms its consensus. The SMCWG will consult other technical communities when and
+as necessary.
+
+The problem to be addressed by the working group is the absence of consistent
+and audited validation practices used by CAs in establishing the identity of the
+subject (if present) and verifying that the subscriber controls the email
+address. While there are methods for validating control of a domain that can be
+adopted from TLS certificates, there are currently no standard requirements for
+validating control of email addresses. Methods for validating identities in TLS
+certificates also exist and should be leveraged where possible, as well as other
+identity validation standards common in the industry.
+At least one method to efficiently validate an email address as well as
+developing a consistent profile for S/MIME certificates in order to facilitate
+technical interoperability across the Internet are also to be addressed by
+the SMCWG.
+
+## Chartering of the S/MIME Certificate Working Group
+
+A Chartered Working Group (“SMCWG”) is created to perform the activities as
+specified in this Charter, subject to the terms and conditions of the
+[CA/Browser Forum Bylaws](https://cabforum.org/bylaws/) and [Intellectual
+Property Rights (IPR) Policy](https://cabforum.org/ipr-policy/), as such
+documents may change from time to time. This charter for the S/MIME Certificate
+Working Group has been created according to CAB Forum Bylaw 5.3.1. In the event
+of a conflict between this Charter and any provision in either the Bylaws or
+the IPR Policy, the provision in the Bylaws or IPR Policy SHALL take
+precedence. The definitions found in the Forum’s Bylaws SHALL apply to
+capitalized terms in this Charter.
+
+### 1. Scope
+
+The authorized scope of the SMCWG SHALL be to discuss, adopt, and maintain
+policies, frameworks, and sets of standards related to the issuance and
+management of S/MIME certificates by third-party CAs under a publicly trusted
+root, limited as follows:
+
+- Baseline verification of control over email addresses, including those used
+  by a natural person or a legal entity, or used by automated systems such as
+  for mailing lists
+- Identity validation for natural persons and legal entities
+- Key management and certificate lifecycle (subject to coordination with other
+  Forum CWGs to ensure consistency and avoid redundancy)
+- Certificate profiles for S/MIME certificates and Issuing CA certificates
+  (including the appropriateness of extensions and when those extensions should
+  be present)
+- CA operational practices, physical/logical security, etc.
+
+The SMCWG SHALL exercise caution to ensure that its work product does not impede
+the issuance of certificates with other EKU types.
```

Comment:
I'm worried about this statement, because a profile for intermediate certificates that precludes the use of `id-kp-serverAuth` may be interpreted as impeding the issuance of certificate with other EKU types.

---

Certificate profiles for the SMCWG SHALL be constrained to those certificates technically capable of issuing certificates with the emailProtection (OID: `1.3.6.1.5.5.7.3.4`) Extended Key Usage (EKU).

---

This still leaves 'technically capable' somewhat ambiguous, but at least then the answer for any impedance, if any, is separate roots, rather than suggesting that such requirements cannot be adopted at all.

Reply:
I thought the point of this statement was to ensure that S/MIME requirements don't prevent CAs from issuing certificates that are also used for other purposes, such as electronic signatures. If so, the proposal doesn't appear to retain that intent.

Reply:
@wthayer I tried to tweak this a little, in https://github.com/sleevi/cabforum-docs/pull/9

My understanding, and what I'm trying to capture here, is that this is trying to prevent the SMCWG from impinging on other CWGs (such as the SCWG).

That is, it's not about ensuring that a given sub-CA can issue whatever it wants, but that the scope of the SMCWG shall be focused to only those certificates that are technically capable.

In terms of a SHALL requirement, the existing language is ambiguous. What constitutes exercising caution? Is the SMCWG forbidden from taking up a TLS profile? Or could it just "exercise caution" and do so anyways? This tries to make it clear such an activity is out of scope.

That said, I agree that it doesn't and cannot prevent the SMCWG from defining a Root Certificate profile that is different from the SCWG's Root Certificate profile. The current language doesn't prevent that either, though.

I mention this, because the Browser Alignment ballot (which tries to align with existing root store policies) already has language regarding [EKUs that may be permitted](https://github.com/cabforum/documents/compare/master...sleevi:2019-10-Browser_Alignment#diff-7f6d14a20e7f3beb696b45e1bf8196f2R1559), and the current language may be seen as precluding from being adopted in this charter.

Reply:
@sleevi sounds like we hold different beliefs on the purpose of that sentence. Again, I think it is a weak attempt to instruct the SMCWG not to institute rules for S/MIME that revent the use of the same cert for other uses - primarily electronic signatures. If so, the #9 language doesn't help. Retaining the existing language but replacing the SHALL with a SHOULD would do that (but the meaning would still be unclear).

Having said that, I'm arguing a position that I don't really care about. I'm happy with the #9 language as long as it doesn't later become a sticking point. @timfromdigicert WDYT?

Reply:
I think there's some in sleevi#9, but it needs some tweaks.  I need to go over it more carefully.